src/components/PerfHead/PerfHead.astro

```diff
@@ -1,7 +1,22 @@
 ---
```

Comment:
Add tests for this file?

Reply:
That also triggers the question if the `fonts` and `preConnectOrigins` should be passed as props?